.ado/azure-pipelines.yml

```diff
@@ -93,6 +93,15 @@ jobs:
       - checkout: self
         persistCredentials: true
 
+      - bash: echo Test 1
+        condition: and(succeeded(), eq($(relative_directory), 'macos'))
+
+      - bash: echo Test 2
+        condition: and(succeeded(), eq($(relative_directory), macos))
+
+      - bash: echo Test 3
```

Comment:
what are we testing here?

Reply:
I'm going to remove this code, I just needed to test what syntax would work for certain conditions. I'm going to mark this PR as a draft until I figure it out :)